docs/BOOTING.md

```diff
@@ -136,6 +136,23 @@ in grub.cfg with graphical GRUB menu to get the device to boot again.
 3. General kernel parameters may be adjusted with `set_global dom0_extra_args "$dom0_extra_args OPTION1=VAL1 OPTION2 "`.
    They will be added to kernel cmdline.
 
+As an example of using serial port and VGA as a console the following kernel arguments
+should be used:
+
+```
+set_global dom0_extra_args "$dom0_extra_args console=ttyS0,115200n8 console=tty0"
+```
+
+defines that opening ``/dev/console`` will get you the current foreground virtual
+console, and kernel messages will appear on both the VGA console and the 1st serial
+port (ttyS0 or COM1) at 115200 baud.
+
+Details about serial console kernel arguments can be also seen [here](https://www.kernel.org/doc/html/v6.1/admin-guide/serial-console.html).
```

Comment:
Maybe, "Details about serial console kernel arguments for Linux in general are available on official kernel documentation.", with the hyperlink on "official kernel documentation"?

Reply:
Done